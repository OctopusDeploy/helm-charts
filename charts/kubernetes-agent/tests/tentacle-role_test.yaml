suite: "tentacle permissions"
templates:
  - templates/tentacle-role.yaml
tests:
  - it: "should match snapshot"
    asserts:
      - matchSnapshot: {}