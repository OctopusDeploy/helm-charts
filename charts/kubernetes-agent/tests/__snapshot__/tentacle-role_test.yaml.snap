should match snapshot:
  1: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: Role
    metadata:
      name: octopus-agent-tentacle-role
      namespace: NAMESPACE
    rules:
      - apiGroups:
          - '*'
        resources:
          - pods
          - pods/log
          - configmaps
          - secrets
        verbs:
          - '*'
